### ISSUE TITLE
The public GPG key has expired 

### ISSUE THREAD

**Issue by Sentient07** [NVIDIA/nvidia-docker#1524]
Hello, 

It seems like the key for `nvidia-container-runtime` has expired on 2021/06/16. Similar to issue #1081 , but the [comment](https://github.com/NVIDIA/nvidia-docker/issues/1081#issuecomment-533717708) doesn't work anymore. Here is my (relevant) traceback,

```


$ sudo apt-get update
--
  | Get:1 file:/var/cuda-repo-ubuntu2004-11-3-local  InRelease
  | Ign:1 file:/var/cuda-repo-ubuntu2004-11-3-local  InRelease
  | Get:2 file:/var/cuda-repo-ubuntu2004-11-4-local  InRelease
  | Ign:2 file:/var/cuda-repo-ubuntu2004-11-4-local  InRelease
  | Get:3 file:/var/nvidia-driver-local-repo-ubuntu1804-470.57.02  InRelease
  | Ign:3 file:/var/nvidia-driver-local-repo-ubuntu1804-470.57.02  InRelease
  | Get:4 file:/var/cuda-repo-ubuntu2004-11-3-local  Release [564 B]
  | Hit:5 http://fr.archive.ubuntu.com/ubuntu focal InRelease
  | Get:4 file:/var/cuda-repo-ubuntu2004-11-3-local  Release [564 B]
  | Hit:6 http://fr.archive.ubuntu.com/ubuntu focal-updates InRelease
  | Hit:7 http://fr.archive.ubuntu.com/ubuntu focal-backports InRelease
  | Hit:8 http://fr.archive.ubuntu.com/ubuntu focal-security InRelease
  | Get:9 file:/var/cuda-repo-ubuntu2004-11-4-local  Release [564 B]
  | Get:9 file:/var/cuda-repo-ubuntu2004-11-4-local  Release [564 B]
  | Get:10 file:/var/nvidia-driver-local-repo-ubuntu1804-470.57.02  Release [564 B]
  | Get:10 file:/var/nvidia-driver-local-repo-ubuntu1804-470.57.02  Release [564 B]
  | Hit:11 https://nvidia.github.io/libnvidia-container/experimental/ubuntu18.04/amd64  InRelease
  | Hit:12 https://download.docker.com/linux/ubuntu focal InRelease
  | Get:13 https://nvidia.github.io/nvidia-container-runtime/experimental/ubuntu18.04/amd64  InRelease [1149 B]
  | Hit:14 https://nvidia.github.io/libnvidia-container/stable/ubuntu18.04/amd64  InRelease
  | Hit:15 https://nvidia.github.io/nvidia-container-runtime/stable/ubuntu18.04/amd64  InRelease
  | Hit:16 https://nvidia.github.io/nvidia-docker/ubuntu18.04/amd64  InRelease
  | Hit:17 https://linux.dell.com/repo/community/openmanage/950/focal focal InRelease
  | Err:13 https://nvidia.github.io/nvidia-container-runtime/experimental/ubuntu18.04/amd64  InRelease
  | The following signatures were invalid: EXPKEYSIG 6ED91CA3AC1160CD NVIDIA CORPORATION (Open Source Projects) <cudatools@nvidia.com>
  | Fetched 1149 B in 2s (709 B/s)
  | Reading package lists... Done
  | W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://nvidia.github.io/nvidia-container-runtime/experimental/ubuntu18.04/amd64  InRelease: The following signatures were invalid: EXPKEYSIG 6ED91CA3AC1160CD NVIDIA CORPORATION (Open Source Projects) <cudatools@nvidia.com>
  | W: Failed to fetch https://nvidia.github.io/nvidia-container-runtime/experimental/ubuntu18.04/amd64/InRelease  The following signatures were invalid: EXPKEYSIG 6ED91CA3AC1160CD NVIDIA CORPORATION (Open Source Projects) <cudatools@nvidia.com>
  | W: Some index files failed to download. They have been ignored, or old ones used instead.


```

**Comment by elezar** [comment]
Hi @Sentient07. 

There is a known issue with our `experimental` GPG key that we are working on resolving. Please remove the `experimental` repos from your apt lists for the time being. These are currently behind the `stable` repos in any case.

**Comment by Sentient07** [comment]
Hello @elezar thanks for the reply, is there a stable version of `nvidia-container-runtime` I'm unable to find this myself, can you please point me to the alternative (Stable) one? Thank you

**Comment by elezar** [comment]
Sorry, I mean you need to remove / uncomment the entries from the `.list` file `/etc/apt/sources.list.d/`. If you could provide me with the contents of the files there, I can point out the exact lines to remove.

**Comment by elezar** [comment]
See for example https://github.com/NVIDIA/nvidia-container-runtime/issues/144

**Comment by elezar** [comment]
Hi @Sentient07

This issue should be resolved now. Could you please check that you are able to update the repos (including the experimental ones) and close the issue accordingly.

**Comment by Sentient07** [comment]
Hello, Yes, I can confirm it's fixed. Thanks!

**Comment by aastha01chauhan** [comment]
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://nvidia.github.io/libnvidia-container/stable/ubuntu18.04/amd64  InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY DDCAE044F796ECB0
W: Failed to fetch https://nvidia.github.io/libnvidia-container/stable/ubuntu18.04/amd64/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY DDCAE044F796ECB0
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by aastha01chauhan** [comment]
> W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://nvidia.github.io/libnvidia-container/stable/ubuntu18.04/amd64 InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY DDCAE044F796ECB0 W: Failed to fetch https://nvidia.github.io/libnvidia-container/stable/ubuntu18.04/amd64/InRelease The following signatures couldn't be verified because the public key is not available: NO_PUBKEY DDCAE044F796ECB0 W: Some index files failed to download. They have been ignored, or old ones used instead.

Getting this error, can someone suggest what to do?

**Comment by aastha01chauhan** [comment]
![image](https://user-images.githubusercontent.com/63860869/221373494-abd21bcc-f2ac-4316-9889-67486479f043.png)
